src/Event/EventMessageNotifier.php

```diff
@@ -74,7 +74,7 @@ public function onEventCreated(EventEvent $event): void
 
     public function onEventCancelled(EventEvent $event): void
     {
-        if (!$event->isCoalitionsEvent() && (!$event instanceof CommitteeEventEvent || !$event->getCommittee())) {
+        if ((!$event instanceof CommitteeEventEvent || !$event->getCommittee()) && !$event->needSendMessage()) {
```

Comment:
it seems that `needNotifyForCancellation` is not used, maybe you should use it here?